referencia/webpay/README.md

```diff
@@ -1788,3 +1788,100 @@ username  <br> <i> xs:string </i> | Nombre de usuario, del cliente, en el sistem
 Nombre  <br> <i> tipo </i> | Descripción
 ------   | -----------
 result  <br> <i> xs:boolean </i> | Indica si la eliminación se realizó correctamente o no.
+
+## Otros Servicios Webpay OneClick Mall
```

Comment:
I would not create a new sub-section. Deferred capture can just come after the "Reversar o Anular..." sub-sub-section above (before the documentation you are adding)

Reply:
Ok